src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -25,24 +25,41 @@ EOF
 }
 
 function clone () {
-    curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    if [ $TEST_TYPE == "local" ]; then
+        curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    elif [ $TEST_TYPE == "kube" ]; then
+        kubectl exec -it $CODEWIND_POD_ID -- curl -H "Content-type: application/json" -d '{ "projectName": "'$1'" , "parentPath": "'$2'", "url": "'$3'" }' "https://localhost:9191/api/v1/projects" --insecure
```

Comment:
Do we also need to use the curl to clone projects to workspace for kube case? Think we design to use Devfile, then projects will be downloaded to workspace automatically.

Reply:
same as above. will remove from dev file and clone like local because of portal initialization in chart